release-notes/v1.2.0.yaml

```diff
@@ -0,0 +1,146 @@
+  Fixed Delay in SecurityPolicy change propagation for HTTPRoute when using targetSelectors
+  Fixed JSONPath not being correctly translated to JSONPatch paths
+  Fixed allowing an empty slowStart value when using LeastRequest
+  Fixed Backends which should be rejected are still used as an HTTPRoute's destination
+  Fixed timeout settings originating from the route being lost when translating the backend traffic policy
+  Fixed Backend resources not receiving status updates
+  Fixed active health checks requiring the expectedStatuses field to function correctly
+  Fixed HTTPHeaderFilter processing not correctly supporting multiple header values
+  Fixed reconciling multiple ReferenceGrants within the same namespace
+  Fixed unwanted / appearing in the Path when using Prefix Rewrites
+  Fixed creation of SecurityPolicy with targetSelectors fails
+  Fixed incorrect gateway being selected as the HTTPRoute parent
+  Fixed override issues for EnvoyExtensionPolicy
+  Fixed nil pointer error when translating hash load balancing
+  Fixed ratelimit does not work across multiple GatewayClasses
+  Fixed upstream mTLS only works for HTTPS listeners
+  Fixed nil pointer if backedtls.minVersion is set but backedtls.maxVersion is not
+  Fixed empty connection limits causing xDS rejection
+  Fixed rate limiting not working with both headers and CIDR matches
+  Fixed EDS not updating when deployments were created after services
+  Fixed RBAC issue for deleting infrastructure resources
+  Fixed gateways never reaching ready/programmed status when running Envoy as a Daemonset
+  Fixed rate limit deployment ignoring pod labels and annotation merges
+  Fixed the API Server receives unnecessary requests
+  Fixed terminating envoy pods don't respond with "Connection: close" (H1) or GOAWAY(H2) on shutdown, switch to an immediate drain strategy
+  Fixed ratelimit statsd not working
+  Fixed not generating selector of deployment/daemonset based on the custom label configuration of EnvoyProxy
+  Fixed egctl experimental translate using an incorrect namespace
+  Fixed reconciliation not being triggered for Secret updates referenced by a BackendTLSPolicy
+  Fixed xDS translation failure when WASM HTTP code source was configured without an SHA
+  Fixed HTTPRoute status displaying only one parent when targeting multiple gateways from different GatewayClasses
+  Fixed Route with multiple parents having an incorrect namespace in the parentRef status
+  Fixed BackendTlsPolicy specifying multiple targetRefs for the same service, to work
+
+# Enhancements that improve performance.
+performance improvements: |
+  Optimize memory usage by only storing distinct resources
+
+# Other notable changes not covered by the above sections.
+Other changes: |
+  Upgraded Envoy to v1.32.1
+  Reduced the amount of configuration logging, and make it line-delimited friendly
+  Made watching alpha CRDs optional, so that gateway-api upgrade don't break envoy gateway
```

Comment:
Made watching alpha CRDs optional, so that Envoy Gateway can run with older Gateway Api versions